doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml

```diff
@@ -1,1129 +1,1921 @@
+      </summary>
+      <returns>
+        <see langword="true" /> if the <see cref="T:Microsoft.Data.SqlClient.SqlConnectionStringBuilder" /> contains an element that has the specified key; otherwise, <see langword="false" />.
+      </returns>
+      <remarks>
+        <para>
+          Because the <see cref="T:Microsoft.Data.SqlClient.SqlConnectionStringBuilder" /> contains a fixed-size collection of key/value pairs, the <see cref="M:System.Data.Common.DbConnectionStringBuilder.ContainsKey" /> method determines only if a particular key name is valid.
+        </para>
+      </remarks>
+      <example>
+        <para>
+          The following example creates a <see cref="T:Microsoft.Data.SqlClient.SqlConnectionStringBuilder" /> instance, sets some of its properties, and then tries to determine whether various keys exist within the object by calling the <b>ContainsKey</b> method.
+        </para>
+        <!-- SqlConnectionStringBuilder_ContainsKey -->
+        <code language="c#">
+          using System;
+          using System.Data;
+          using Microsoft.Data.SqlClient;
+          
+          class Program
+          {
+              static void Main()
+              {
+                  SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(GetConnectionString());
+                  Console.WriteLine("Connection string = " + builder.ConnectionString);
+          
+                  // Keys you have provided return true.
+                  Console.WriteLine(builder.ContainsKey("Server"));
+          
+                  // Comparison is case insensitive, and synonyms
+                  // are automatically converted to their "well-known"
+                  // names.
+                  Console.WriteLine(builder.ContainsKey("Database"));
+          
+                  // Keys that are valid but have not been set return true.
+                  Console.WriteLine(builder.ContainsKey("Max Pool Size"));
+          
+                  // Keys that do not exist return false.
+                  Console.WriteLine(builder.ContainsKey("MyKey"));
+          
+                  Console.WriteLine("Press Enter to continue.");
+                  Console.ReadLine();
+              }
+          
+              private static string GetConnectionString()
+              {
+                  // To avoid storing the connection string in your code,
+                  // you can retrieve it from a configuration file. 
+                  return "Server=(local);Integrated Security=SSPI;" +
+                         "Initial Catalog=AdventureWorks";
+              }
+          }
+        </code>
+        <para>
+          The example displays the following output in the console window:
+        </para>
+        <code>
+          Connection string = Data Source=(local);Initial Catalog=AdventureWorks;Integrated Security=True
+          True
+          True
```

Comment:
looks like something went wrong here

Reply:
Nope, that's about right. That is supposed to show the output the example code and it outputs the connection string then outputs whether it contains a handful of keys. So the expected output is a bunch of trues. But, it's always good to double check stuff like that :+1: